venues/thecvf.com/ECCV/2020/Conference/python/tpms-get-pdfs.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import print_function
+import requests
+import openreview
+import argparse
+import os
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--baseurl', help="base url")
+parser.add_argument('--username')
+parser.add_argument('--password')
+args = parser.parse_args()
+
+client = openreview.Client(baseurl=args.baseurl, username=args.username, password=args.password)
+print ('connecting to {0}'.format(client.baseurl))
+
+submission_invitation = 'thecvf.com/ECCV/2020/Conference/-/Submission'
```

Comment:
well you are using the original invitation, but there is not details.original, does this script work?

Reply:
Made changes to use the original notes.